tests/model_registry/conftest.py

```diff
@@ -303,3 +316,61 @@ def generated_schema(model_registry_instance_rest_endpoint: str) -> Any:
         uri="https://raw.githubusercontent.com/kubeflow/model-registry/main/api/openapi/model-registry.yaml",
         base_url=f"https://{model_registry_instance_rest_endpoint}/",
     )
+
+
+def pytest_sessionstart(session: pytest.Session) -> None:
+    """Downloads external tests into the local project structure before test execution."""
+    if session.config.option.model_registry_upstream:
+        LOGGER.info("pytest_sessionstart is running")
+        os.makedirs(UPSTREAM_TESTS_DIR, exist_ok=True)
+        for file in FILES:
+            os.system(f"curl -o {UPSTREAM_TESTS_DIR}/{file} {REPO_API_URL}/{FILE_PATH}/{file}")
```

Comment:
why not use `run_command`?

Reply:
wouldn't it be better to use subprocess.run() here?